manifests-maistra/1.1.0/servicemeshcontrolplanes.crd.yaml

```diff
@@ -16,25 +16,30 @@ spec:
     status: {}
   version: v1
   additionalPrinterColumns:
-  - JSONPath: .status.conditions[?(@.type=="Ready")].status
-    name: Ready
+  - name: Ready
+    description: How many of the total number of member namespaces are
+      configured
+    type: string
+    JSONPath: '.status.annotations.readyComponentCount'
+  - JSONPath: .status.conditions[?(@.type=="Ready")].reason
+    name: Status
     description: Whether or not the control plane installation is up to date and ready
       to handle requests.
     type: string
-  - JSONPath: .status.conditions[?(@.type=="Ready")].message
-    name: Status
-    description: The status of the control plane installation.
+  - name: Template
+    description: The configuration template to use as the base.
     type: string
-    priority: 1
-  - JSONPath: .status.conditions[?(@.type=="Reconciled")].status
-    name: Reconciled
-    description: Whether or not the control plane installation is up to date with
-      the latest version of this resource.
+    JSONPath: .status.lastAppliedConfiguration.template
+  - name: Version
+    description: The actual current version of the control plane installation.
     type: string
-    priority: 1
-  - JSONPath: .status.conditions[?(@.type=="Reconciled")].message
-    name: Reconciliation Status
-    description: The status of the reconciliation process, if the control plane is
-      not up to date with the latest version this resource.
+    JSONPath: .status.lastAppliedConfiguration.version
+  - name: Age
+    description: The age of the object
+    type: date
+    JSONPath: .metadata.creationTimestamp
+  - name: Image HUB
```

Comment:
If you're going to have hub, why not tag.  I think tag is probably more useful, as it implies an install version.

I'm leery about adding hub, as we might move to using image streams in the future.  Maybe that would imply a "local" hub.

Regardless, would `Image Registry` be a better term?

Reply:
I initially called this column "IMAGE REGISTRY", as it _is_ a better term. But since we use "hub" in the smcp.spec, I thought we should keep it consistent with the spec.

I'll also add the TAG.